docker-compose.yml

```diff
@@ -177,3 +178,68 @@ services:
   redis:
     image: redis
     restart: always
+
+
+
+
+
+  zookeeper:
+    image: "zookeeper:3.4"
+    container_name: zookeeper
```

Comment:
Custom container names were used to easily access the container instance with 'docker exec' during mutual authentication adaptation phase. Since they are not necessary now, they can be removed.